PsfLauncher/PsfPowershellScriptRunner.h

```diff
@@ -269,6 +269,10 @@ class PsfPowershellScriptRunner
 					SSWrapper = file.path();
 				}
 			}
+			if (!std::filesystem::exists(SSWrapper))

```

Comment:
NIT: Move this check up to Initialize.  Sooner we check for the existence of a required file, the better.

Reply:
Moving this to intiailize would require additional evaluation for wrapperScript path while executing the powershell script(or adding one more class attribute for storing wrapper script path). To avoid re-duplication we remained it this way. Let me know your opinion.

Reply:
Ic ic.  Looking at it, I can see that moving this would be a bit hard.  Keep it here, but moving it up in the future would be awesome.